client/src/App.jsx

```diff
@@ -92,7 +87,7 @@ class App extends React.Component {
 
       //probably should have just pushed photoArr data and mapped that
```

Comment:
this is much better than saving an array of elements!! that being said you don't need to store duplicate data, you can just use the current page and photo array state you already have those in state to render the appropriate 5 photos and a rerender will be triggerered on every page change, you might be unnecessarily triggering rerenders at the moment.